UnityProject/Assets/Scripts/Tilemaps/Behaviours/Objects/AirVent.cs

```diff
@@ -53,6 +53,11 @@ private void CheckAtmos()
 				return;
 			}
 
+			if (metaNode.GasMix.Pressure == 0)
```

Comment:
What does this do?

Reply:
yeah, it was dividing by zero so I need to think of a better thing for it, in case it is zero

Reply:
fixed